addons/godot-xr-tools/interactables/interactable_handle.gd

```diff
@@ -22,6 +22,20 @@ extends XRToolsPickable
 ## Distance from the handle origin to auto-snap the grab
 @export var snap_distance : float = 0.3
 
+@export_group("Optional attach Hand")
+## this latches the grabbing Hand onto the Interactable
```

Comment:
This block of '##' (API Documentation Comments) produces some really funky online documentation.
![image](https://github.com/GodotVR/godot-xr-tools/assets/1863707/d746041b-14ab-448f-a60e-13ef88f8b5c2)

If this is just documentation for developers, you may want to use the '#' style of comments.

Reply:
thanks for taking a look at this, wasnt aware that this would actualy produce this kind of mess.
gonna look into it and remove unecassary comments.